src/components/FieldInput.tsx

```diff
@@ -0,0 +1,36 @@
+import { useField } from "react-final-form";
+
+const TextInput = ({
+  nameLabel,
```

Comment:
It should just named `label`
it should just named `name`
it should just named `type`
it should just named `placeholder`

you don't need to prefix `name`